views/admin.handlebars

```diff
@@ -21,5 +21,30 @@
         {{/each}}
         </tbody>
     </table>
-</div>
+
+
+    <h2>Inventory by Warehouse</h2>
+    <table class="table">
+        <thead>
+        <tr>
+            <th scope="col">Product ID</th>
```

Comment:
Could you do this so each table is a separate warehouse instead? If you take a look at listOrder is should be similar.